modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java

```diff
@@ -669,6 +669,16 @@ public ChildT distribution(IgniteDistribution distribution) {
             return self();
         }
 
+        /** {@inheritDoc} */
+        @Override
+        public ChildT addKeyColumn(String name, NativeType type) {
+            columns.add(new ColumnDescriptorImpl(
+                    name, true, true, columns.size(), columns.size(), type, DefaultValueStrategy.DEFAULT_NULL, null
```

Comment:
why 'nullable' is true?

Reply:
`The primary key constraint specifies that a column or columns of a table can contain only unique (non-duplicate), nonnull values. Technically, PRIMARY KEY is merely a combination of UNIQUE and NOT NULL.`

Reply:
Thanks, fixed.